osu.Game.Rulesets.Osu/Skinning/Default/PlaySliderBody.cs

```diff
@@ -44,7 +44,6 @@ private void load(ISkinSource skin, DrawableHitObject drawableObject)
 
             SnakingOut.BindTo(configSnakingOut);
 
-            BorderSize = skin.GetConfig<OsuSkinConfiguration, float>(OsuSkinConfiguration.SliderBorderSize)?.Value ?? 1;
```

Comment:
I went ahead and removed this in accordance with https://github.com/ppy/osu/discussions/25057#discussioncomment-7259535, and as to not clutter the logic in `LegacySliderBody` any further.